x-pack/plugins/alerting/server/routes/get_query_delay_settings.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IRouter } from '@kbn/core/server';
+import { ILicenseState } from '../lib';
+import { AlertingRequestHandlerContext, INTERNAL_BASE_ALERTING_API_PATH } from '../types';
+import { verifyAccessAndContext, RewriteResponseCase } from './lib';
+import { API_PRIVILEGES, RulesSettingsQueryDelay } from '../../common';
+
+const rewriteBodyRes: RewriteResponseCase<RulesSettingsQueryDelay> = ({
+  createdBy,
+  updatedBy,
+  createdAt,
+  updatedAt,
+  ...rest
+}) => ({
+  ...rest,
+  created_by: createdBy,
+  updated_by: updatedBy,
+  created_at: createdAt,
+  updated_at: updatedAt,
+});
+
+export const getQueryDelaySettingsRoute = (
```

Comment:
should we be versioning these routes?

Reply:
Resolved in this commit a105003ab3f52214800ee7cd390211a981d6cabe